data/json/recipes/food/baking.json

```diff
@@ -65,5 +65,28 @@
       [ [ "salt", 2 ] ],
       [ [ "cinnamon", 2 ] ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "baklava",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_SNACK",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "2 hours",
+    "batch_time_factors": [ 80, 1 ],
+    "book_learn": [ [ "cookbook_turkish", 2 ] ],
+    "qualities": [ { "id": "COOK", "level": 2 }, { "id": "OVEN", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 20, "LIST" ] ] ],
+    "components": [
+      [ [ "walnut_shelled", 4 ] ],
+      [ [ "honey_bottled", 1 ] ],
```

Comment:
I'd make this honey or maple syrup here.

Reply:
I couldn't find "honey" anywhere as a food item by itself, only as a material but I'm not sure how different that is.
![image](https://user-images.githubusercontent.com/57331165/233211182-f3471908-c741-4455-9641-5118e859f275.png)

Reply:
Sorry I meant add maple syrup as an option as well as the bottled honey you have already there.

Reply:
I see! I added maple_sap as an option!